images/openshift-golang-builder.Dockerfile

```
@@ -1,4 +1,4 @@
-FROM openshift/ose-base:rhel8.2.els.rhel
+FROM registry.redhat.io/ubi8:8.3
```

we don't have to change again if it's:
```suggestion
FROM registry.redhat.io/ubi8:latest
```